src/anomalib/models/padim/config.yaml

```diff
@@ -36,9 +36,12 @@ model:
 
 post_processing:
   normalization_method: min_max # <null, min_max, cdf>
-  threshold_method: adaptive # options: [adaptive, manual]
-  manual_image_threshold: null
-  manual_pixel_threshold: null
+  image_threshold:
+    class_path: AdaptiveScoreThreshold # see anomalib.utils.metrics.thresholding for more options
+    init_args: null
```

Comment:
when there is no `init_args` to pass, do we still need to pass it as `null`. What is the behaviour when we just do the following:

```
  image_threshold:
    class_path: AdaptiveScoreThreshold # see anomalib.utils.metrics.thresholding for more options
```